tls/README.md

```diff
@@ -1242,3 +1242,52 @@ Sending test string
 Sent (0): Testing 1, 2 and 3
 Read (0): Testing 1, 2 and 3
 ```
+
+
+## TLS Example with PK Callbacks and optionally Async
+
+See `server-tls-pkcallback.c` and `client-tls-pkcallback.c`.
+
+The top of both files have an optional build setting to gate use of ECC/RSA and TLS v1.2 or TLS v1.3:
```

Comment:
why not highest and then ability to downgrade? (ie `23()` )

Reply:
The PK callback paths for TLS v1.2 and v1.3 are totally different. I wanted to have a build-time option to test each.